lib/ansible/modules/cloud/azure/azure_rm_securitygroup.py

```diff
@@ -548,7 +548,7 @@ def __init__(self):
                                                    supports_check_mode=True)
 
     def exec_module(self, **kwargs):
-        self.client = self.get_mgmt_svc_client(NetworkManagementClient)
+        self.client = self.network_client
```

Comment:
why not replacing the self.client to self.network_client?

Reply:
for easy..